packages/leva/src/components/Leva/LevaRoot.tsx

```diff
@@ -114,13 +115,15 @@ export function LevaRoot({ store, hidden = false, theme, collapsed = false, ...p
 
   return (
     <ThemeContext.Provider value={themeContext}>
-      <LevaCore
-        store={store}
-        {...props}
-        toggled={computedToggled}
-        setToggle={computedSetToggle}
-        rootClass={themeContext.className}
-      />
+      <RadixTooltip.Provider>
```

Comment:
I've finally instantiated the `<RadixTooltip.Provider>` once in the `LevaRoot` component